cmd/controller-bootstrap/command/generate.go

```diff
@@ -14,19 +14,97 @@
 package command
 
 import (
+	"bytes"
+	"context"
+	"fmt"
 	"github.com/spf13/cobra"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
```

Comment:
external imports are separated by a new line.
```suggestion
	"bytes"
	"context"
	"fmt"
	"io/ioutil"
	"os"
	"path/filepath"
	"strings"
	"text/template"
	
	"github.com/spf13/cobra"
```